### DISCUSSION TITLE
Consolidating files from previous versions

### DISCUSSION THREAD

**Discussion by pandamega** [nianeyna/ao3downloader#157] [Q&A]
Hello! I first used this program in 2022 and the downloaded files were named "fandom - author - title." I have just downloaded my bookmarks page again on a different computer and see that the files are now named "work ID - Title - author." I now have a bit of a problem because I would like to consolidate both download folders, but I can't easily remove the duplicates because they have different filenames. I don't want to get rid of the older downloads because many of my favorite fics have been deleted since 2022. AO3 downloader can still recognize the older versions of the fics so as not to download them again, so I was wondering if there was a quick way to either find the duplicates or rename the old files? Thank you!

**Comment by nianeyna** [comment]
I don't think you could configure ao3downloader itself to help with this, but there should be some other programs available that can identify duplicate files, depending on your operating system
